Fix lack of return in non-void function.

https://sourceforge.net/tracker/?group_id=213712&atid=1026806&aid=3454858
https://sourceforge.net/p/gelide/bugs/16/

Change 5 of 5 from
https://sourceforge.net/p/gelide/code/ci/5fd424b9240de4794cb75cda81aa0f84ec4326f8/
--- src/ui/dialog_emulator_launcher.cpp
+++ src/ui/dialog_emulator_launcher.cpp
@@ -245,7 +245,10 @@ int CDialogEmulatorLauncher::run(CSystem* p_system, CGame* p_game){
 		return Gtk::Dialog::run();
 	}
 	else
+	{
 		this->response(Gtk::RESPONSE_CLOSE);
+		return Gtk::RESPONSE_CLOSE;
+	}
 }
 
 void CDialogEmulatorLauncher::loadConfig(void){
